ilastik/applets/layerViewer/layerViewerGui.py

```diff
@@ -69,7 +69,7 @@ def __call__(cls, *args, **kwargs):
         instance._after_init()
         return instance
 
-class LayerViewerGui(QWidget):
+class LayerViewerGui(QWidget, metaclass=LayerViewerGuiMetaclass):
     """
```

Comment:
FWIW, it looks like `six` has a solution `with_metaclass` that would let us keep Py2 and Py3 support simultaneously.